x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.ts

```diff
@@ -47,13 +47,10 @@ export const buildBulkBody = (
     ...buildSignal([mergedDoc], rule, reason),
     ...additionalSignalFields(mergedDoc),
   };
-  const event = buildEventTypeSignal(mergedDoc);
```

Comment:
@machadoum lets drop this change. The tests are gonna fail again `ciGroup11` and this is why. You can just undo it.